tests/test_lib.rs

```diff
@@ -49,7 +49,6 @@ fn load_tests() -> Vec<TestCase> {
         .collect()
 }
 
-#[cfg(not(feature = "wasm"))]
```

Comment:
Was this needed before due to these changes?

https://github.com/Bestowinc/json-logic-rs/blob/85df539c2177d8fa11bd93787534f41224fbb909/tests/test_wasm.rs#L11-L16

Reply:
No, I had added that in because I was trying to get the tests to compile in web assembly, b/c `wasm-pack` has a `wasm-pack test` option that'll compile your existing tests & run them in the WASM target. The `reqwest` package apparently cannot compile to WASM, though, so I was trying to figure out how to get `wasm-pack test` to ignore that bit. I wound up not being able to figure it out, and then forgot about it and left it in.